enterprise-suite/Makefile

```diff
@@ -37,8 +37,21 @@ shell-tests:
 minikube-tests:
 	$(CHART_DIR)/scripts/run-minikube-tests.sh
 
+backend-tests: shell-tests minikube-tests
+
+frontend-tests:
+	$(CHART_DIR)/scripts/run-e2e-tests.sh
+
 # This target required by top-level helm-charts Makefile.  Tests release on Travis
-test: shell-tests minikube-tests
+#
+# The conditional bit is so we can control which set of tests are run given we're only
```

Comment:
I suggested these Makefile (and .travis.yml) changes to Danny.  Not terribly pretty but allows us to define a `test` target for the helm-charts `Makefile` while allowing us to run the backend and frontend tests separately.